build.gradle.kts

```
@@ -17,7 +17,7 @@ dependencies {
     }
 
     // diktat ruleset
-    ktlint("org.cqfn.diktat:diktat-rules:1.0.1") {
+    ktlint("org.cqfn.diktat:diktat-rules:1.0.3-SNAPSHOT") {
```

```suggestion
    ktlint("org.cqfn.diktat:diktat-rules:1.0.2") {
```
Same here, it is and example on usage with released version.